src/main/java/no/unit/nva/model/instancetypes/BookMonograph.java

```diff
@@ -0,0 +1,108 @@
+package no.unit.nva.model.instancetypes;
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import no.unit.nva.model.exceptions.InvalidPageTypeException;
+import no.unit.nva.model.pages.MonographPages;
+import no.unit.nva.model.pages.Pages;
+import nva.commons.utils.JacocoGenerated;
+
+import java.util.Objects;
+
+import static java.util.Objects.nonNull;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+public class BookMonograph implements PublicationInstance {
+
+    private Pages pages;
+    private boolean peerReviewed;
+    private boolean openAccess;
+
+    public BookMonograph() {
+    }
+
+    private BookMonograph(Builder builder) throws InvalidPageTypeException {
+        setPages(builder.pages);
+        setPeerReviewed(builder.peerReviewed);
+        setOpenAccess(builder.openAccess);
+    }
+
+    @Override
+    public Pages getPages() {
+        return pages;
+    }
+
+    @Override
+    public void setPages(Pages pages) throws InvalidPageTypeException {
+        if (nonNull(pages) && !(pages instanceof MonographPages)) {
```

Comment:
What if we  defined PublicationInstance like this:

```
public interface PublicationInstance<P extends Pages> {

    P getPages();

    void setPages(P  pages) throws InvalidPageTypeException;

    void setPeerReviewed(boolean peerReviewed);

    boolean isPeerReviewed();
```

then we could set pages here to be MonographPages.

Reply:
But this is just a suggestion and not for now

Reply:
I think this is an idea worth looking at, and was thinking much the same as I was looking at the tests here and for Journal*. today. I added this to the refactor PublicationInstance task, which I will look at next after merging